llvm/test/CodeGen/Hexagon/fminmax-v67.ll

```diff
@@ -2,7 +2,7 @@
 
 
 ; CHECK-LABEL: t1
```

Comment:
Missing test coverage for other targets?

Reply:
That problem was on the baseline.  But I agree: while we're here, it's a great idea to add a few more `RUN` lines.

Reply:
I have feel knowledge about the flavors of Hexagon.
So which to add?

Reply:
Add new run lines in a separate change

Reply:
Is this patch OK?